crates/polars-python/src/lazyframe/visitor/expr_nodes.rs

```diff
@@ -171,6 +171,21 @@ impl PyStringFunction {
     }
 }
 
+#[pyclass(name = "CategoricalFunction", eq)]
```

Comment:
Can you remove these. I don't think we need those for now.

Reply:
Reverted the whole file, https://github.com/pola-rs/polars/pull/20211/commits/0a83dbef9314ff854a76a6aafb2ed65ea88dcaeb